Runtime/AvatarCreator/Scripts/Managers/AvatarManager.cs

```diff
@@ -288,9 +290,32 @@ public async Task<GameObject> UpdateAsset(AssetType assetType, BodyType bodyType
             {
                 return null;
             }
+            
+#if UNITY_EDITOR
```

Comment:
Why did you make it editor only?

Reply:
No point in showing this error in live environment (This makes extra requests each time)

Reply:
Maybe, if we do asset validation in the future, where we get the correct response immediately, then we can think about it

Reply:
Hmm disagree. 
If somebody tries to use bodyshapes in builds, it should log an error if it's not possible to use them. 
I realize that they would see this when they test in editor but it's still entirely possible that they might just straight cut and paste our elements example and then build and wonder why it's not working.  
It is an extra request but it only happens when you try to use the body shapes element, if they don't want the extra requests then they should remove it.